### DISCUSSION TITLE
Early stage progress

### DISCUSSION THREAD

**Discussion by TheHammerMan00** [makerforgetech/modular-biped#62] [Show and tell]
printing persists and so do I!
![20231021_153635](https://github.com/dmt-labs/modular-biped/assets/33739197/6dd10daa-42c5-4f9e-9c96-a9b10a02d948)

**Comment by danic85** [comment]
Nice work! How did printing go? Any adjustments needed?

**Reply by TheHammerMan00** [reply]
It's going along just fine!  I have the legs and head printed waiting on hardware to arrive so I can start assembly.  I'm working on the body now, and I think Amazon is walking up my driveway now delivering most of the boards.   As far as the printing I may tweak a few minor things.   Some of the walls are very thin and have me worried about durability.  I Think I'll go back and increase the wall thickness on a few parts but I'm sure that will change how things fit together.  I have printed everything in ABS at 30% density so hopefully most of it will hold up for a while.  Only other issue I can think of is probably a setting in my slicer.  It consistently imported the parts in the worst orientation you could.  Lots of moving things around to get the best result.   I'm getting there though. Looking forward to putting things together!
